src/Components/Blueprints/EditBlueprintButton.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+import { Button } from '@patternfly/react-core';
+import { useNavigate } from 'react-router-dom';
+
+import { selectSelectedBlueprintId } from '../../store/BlueprintSlice';
+import { useAppSelector } from '../../store/hooks';
+import { resolveRelPath } from '../../Utilities/path';
+
+export const EditBlueprintButton = () => {
+  const navigate = useNavigate();
+  const selectedBlueprintId = useAppSelector(selectSelectedBlueprintId);
+
+  return (
+    <Button
+      ouiaId="build-images-button"
```

Comment:
oh, the id should be `edit-blueprint-button` or similar, no?

Reply:
Yeah, it should. :eyes:  Great catch, thank you! I've pushed the change